filament/src/PostProcessManager.h

```diff
@@ -71,16 +68,19 @@ struct CameraInfo;
 class PostProcessManager {
 public:
 
-    struct ConstantInfo {
-        std::string_view name;
-        std::variant<int32_t, float, bool> value;
-    };
 
-    struct MaterialInfo {
+    // This is intended to be used only to hold the static material data
+    struct StaticMaterialInfo {
+        struct ConstantInfo {
+            std::string_view name;
+            std::variant<int32_t, float, bool> value;
+        };
         std::string_view name;
         uint8_t const* data;
-        int size;
-        utils::FixedCapacityVector<ConstantInfo> constants = {};
+        size_t size;
+        // the life-time of objects pointed to by this initializer_list<> is extended to the
+        // life-time of the initializer_list
+        std::initializer_list<ConstantInfo> constants;
```

Comment:
What's the advantage of using `std::initializer_list` here instead of `utils::FixedCapacityVector`?

Reply:
good question! This reduces significantly (I should say entirely) the code generated to create the vector. Instead, all the data becomes fully static -- no code generated at all. With the vector, it has to loop through all the items in the list and essentially do a `push_back`.